client/__tests__/reducers/messages.reducer.test.js

```diff
@@ -0,0 +1,67 @@
+import messages, { initialState } from '../../reducers/messages';
+import { Reducer } from 'redux-testkit';
+import * as types from '../../actions/types';
+import expect from 'expect';
+
+describe('group reducer', () => {
+  it('should return an initial state', () => {
+    expect(messages(undefined, {})).toEqual(initialState);
+  });
+  it('should handle GET_GROUP_MESSAGES', () => {
+    const action = {
+      type: types.GET_GROUP_MESSAGES,
```

Comment:
You really need to consider moving all mock data like this to an external file. You have so many cases like this. It will be great taking them out to your external file to encourage your code readability, maintainability and reuse